src/main/java/mate.academy/lib/Injector.java

```diff
@@ -13,32 +13,27 @@
 
 public class Injector {
     private static final Injector INJECTOR = new Injector();
-    private static final Map<Class<?>, Class<?>> INTERFACE_IMPLS = new HashMap<>();
+    private static final Map<Class<?>, Class<?>> INTERFACE_IMPLS = Map.of(
+            FileReaderService.class, FileReaderServiceImpl.class,
+            ProductParser.class, ProductParserImpl.class,
+            ProductService.class, ProductServiceImpl.class
+    );
     private static final Map<Class<?>, Object> INSTANCES = new HashMap<>();
```

Comment:
these fields can have modifiers `private final` without `static` and renamed in camel case style